daphne/src/roles/helper.rs

```diff
@@ -458,42 +514,33 @@ mod tests {
     use futures::StreamExt;
     use prio::codec::ParameterizedDecode;
 
-    use crate::messages::{
-        AggregationJobInitReq, AggregationJobResp, ReportShare, Transition, TransitionVar,
-    };
+    use crate::messages::{AggregationJobInitReq, AggregationJobResp, Transition, TransitionVar};
     use crate::roles::DapHelper;
     use crate::MetaAggregationJobId;
     use crate::{roles::test::TestData, DapVersion};
 
     #[tokio::test]
-    async fn replay_reports_when_continuing_aggregation() {
+    async fn replay_reports_when_continuing_aggregation_draft02() {
         let mut data = TestData::new(DapVersion::Draft02);
         let task_id = data.insert_task(
             DapVersion::Draft02,
-            crate::VdafConfig::Prio2 { dimension: 10 },
```

Comment:
Reviewer note: The VDAF declared here needs to be compatible with the measurement generated by `test.gen_test_report()`. This is Prio3Count, not Prio2.